public-lighting-demo-app/web-demo-app/src/main/java/org/opensmartgridplatform/webdemoapp/infra/platform/SoapRequestHelper.java

```diff
@@ -143,19 +150,33 @@ private HttpComponentsMessageSender createHttpMessageSender() {
         final HttpClientBuilder builder = HttpClients.custom();
         builder.addInterceptorFirst(new ContentLengthHeaderRemoveInterceptor());
         try {
-            final SSLContext sslContext = new SSLContextBuilder()
+            final SSLContext sslContext = SSLContexts.custom()
```

Comment:
Instead of the `SSLContextBuilder` use the same implementation as the `DefaultWebServiceTemplateFactory`.